roles/splunk/vars/RedHat.yml

```diff
@@ -17,3 +17,4 @@ linux_packages:
   - nethogs
   - gdb
   - bind-utils
+firewall_service: firewalld
```

Comment:
You need to add this to the `RedHat8.yml` as well

Reply:
Done in 1acc13b0aa4e7bb55809fb2ed801ffca86b3ba3f